cocos/3d/reflection-probe/reflection-probe-component.ts

```diff
@@ -327,6 +323,12 @@ export class ReflectionProbe extends Component {
         }
     }
 
+    private _handleResize$ (): void {
```

Comment:
Don't add `$` now since we will write a babel plugin to mangle the private variables.